tools/sourcerepo-sync/source_downloader.py

```diff
@@ -0,0 +1,67 @@
+""" Download source repository data from datastore and write to yaml file"""
+from google.cloud import datastore
+
+import os
+import sys
+import argparse
+
+
+def main() -> None:
+  parser = argparse.ArgumentParser(
+      description="Sync source repositories from local yaml to datastore.")
+  parser.add_argument(
+      "--verbose",
+      action=argparse.BooleanOptionalAction,
+      dest="verbose",
+      default=True,
+      help="Display records being operated on")
+  parser.add_argument(
+      "--kind",
+      action="store",
+      dest="kind",
+      default="SourceRepository",
+      help="The datastore kind to operate on")
+  parser.add_argument(
+      "--project",
+      action="store",
+      dest="project",
+      default="oss-vdb-test",
+      help="GCP project to operate on")
+  args = parser.parse_args()
+
+  if args.project == 'oss-vdb-test':
+    file = 'source_test.yaml'
+  elif args.project == 'oss-vdb':
+    file = 'source.yaml'
+  else:
+    print('Invalid project')
+    return
+
+  client = datastore.Client(project=args.project)
+  query = client.query(kind=args.kind)
+  results = list(query.fetch())
+  if args.verbose:
+    print(f'Retrieved {len(results)} sourcerepos')
+  sources = ''
+  for result in results:
```

Comment:
Let's use python objects or dictionaries here + yaml.dump instead of manually writing strings, it'll make a more maintainable in the future if we decide to add more fields.

Reply:
done :)